src/app/GitUI/Editor/FormFindFileGitGrepCommit.Designer.cs

```diff
@@ -66,20 +68,22 @@ private void InitializeComponent()
             lblOptions.TabIndex = 2;
             lblOptions.Text = "&Options:";
             // 
-            // txtSearchFor
+            // cboSearchCommitGitGrep
```

Comment:
But the `label1` could / should be changed into `"&Search for:"` (currently `"&Search:"`).
(adding the comment here because GH does not support commenting unmodified lines)